[2024-04-28T17:35:55.560+0000] {local_task_job_runner.py:120} INFO - ::group::Pre task execution logs
[2024-04-28T17:35:55.636+0000] {taskinstance.py:2073} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: csv_processing_dag.process_csv_files manual__2024-04-28T17:35:53.097157+00:00 [queued]>
[2024-04-28T17:35:55.647+0000] {taskinstance.py:2073} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: csv_processing_dag.process_csv_files manual__2024-04-28T17:35:53.097157+00:00 [queued]>
[2024-04-28T17:35:55.648+0000] {taskinstance.py:2303} INFO - Starting attempt 1 of 2
[2024-04-28T17:35:55.666+0000] {taskinstance.py:2327} INFO - Executing <Task(PythonOperator): process_csv_files> on 2024-04-28 17:35:53.097157+00:00
[2024-04-28T17:35:55.679+0000] {warnings.py:110} WARNING - /home/***/.local/lib/python3.12/site-packages/***/task/task_runner/standard_task_runner.py:61: DeprecationWarning: This process (pid=167) is multi-threaded, use of fork() may lead to deadlocks in the child.
  pid = os.fork()

[2024-04-28T17:35:55.686+0000] {standard_task_runner.py:63} INFO - Started process 169 to run task
[2024-04-28T17:35:55.679+0000] {standard_task_runner.py:90} INFO - Running: ['***', 'tasks', 'run', 'csv_processing_dag', 'process_csv_files', 'manual__2024-04-28T17:35:53.097157+00:00', '--job-id', '100', '--raw', '--subdir', 'DAGS_FOLDER/csv_dags.py', '--cfg-path', '/tmp/tmpsldwfrch']
[2024-04-28T17:35:55.687+0000] {standard_task_runner.py:91} INFO - Job 100: Subtask process_csv_files
[2024-04-28T17:35:55.756+0000] {task_command.py:426} INFO - Running <TaskInstance: csv_processing_dag.process_csv_files manual__2024-04-28T17:35:53.097157+00:00 [running]> on host 4cdb6c1dd19f
[2024-04-28T17:35:55.875+0000] {taskinstance.py:2644} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='***' AIRFLOW_CTX_DAG_ID='csv_processing_dag' AIRFLOW_CTX_TASK_ID='process_csv_files' AIRFLOW_CTX_EXECUTION_DATE='2024-04-28T17:35:53.097157+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='manual__2024-04-28T17:35:53.097157+00:00'
[2024-04-28T17:35:55.877+0000] {taskinstance.py:430} INFO - ::endgroup::
[2024-04-28T17:35:55.899+0000] {python.py:237} INFO - Done. Returned value was: None
[2024-04-28T17:35:55.900+0000] {taskinstance.py:441} INFO - ::group::Post task execution logs
[2024-04-28T17:35:55.910+0000] {taskinstance.py:1205} INFO - Marking task as SUCCESS. dag_id=csv_processing_dag, task_id=process_csv_files, execution_date=20240428T173553, start_date=20240428T173555, end_date=20240428T173555
[2024-04-28T17:35:55.942+0000] {local_task_job_runner.py:240} INFO - Task exited with return code 0
[2024-04-28T17:35:55.959+0000] {taskinstance.py:3482} INFO - 0 downstream tasks scheduled from follow-on schedule check
[2024-04-28T17:35:55.963+0000] {local_task_job_runner.py:222} INFO - ::endgroup::
